src/overlays/actors/ovl_En_Honotrap/z_en_honotrap.c

```diff
@@ -4,15 +4,41 @@
 
 #define THIS ((EnHonotrap*)thisx)
 
+#define HONOTRAP_AT_ACTIVE (1 << 0)
+#define HONOTRAP_AC_ACTIVE (1 << 1)
+#define HONOTRAP_OC_ACTIVE (1 << 2)
+#define HONOTRAP_EYE_OPEN 0
+#define HONOTRAP_EYE_CLOSED 3
```

Comment:
this can prob be an enum, with the middle value included (even if it doesnt get used in the code directly here)